core/src/main/java/com/google/errorprone/scanner/BuiltInCheckerSuppliers.java

```diff
@@ -20,105 +20,8 @@
-import com.google.errorprone.bugpatterns.MissingCasesInEnumSwitch;
-import com.google.errorprone.bugpatterns.MissingFail;
-import com.google.errorprone.bugpatterns.MisusedWeekYear;
-import com.google.errorprone.bugpatterns.ModifyingCollectionWithItself;
-import com.google.errorprone.bugpatterns.NarrowingCompoundAssignment;
-import com.google.errorprone.bugpatterns.NoAllocationChecker;
-import com.google.errorprone.bugpatterns.NonAtomicVolatileUpdate;
-import com.google.errorprone.bugpatterns.NonCanonicalStaticImport;
-import com.google.errorprone.bugpatterns.NonFinalCompileTimeConstant;
-import com.google.errorprone.bugpatterns.NonRuntimeAnnotation;
-import com.google.errorprone.bugpatterns.NullablePrimitive;
-import com.google.errorprone.bugpatterns.NumericEquality;
-import com.google.errorprone.bugpatterns.Overrides;
-import com.google.errorprone.bugpatterns.PackageLocation;
-import com.google.errorprone.bugpatterns.PreconditionsCheckNotNull;
-import com.google.errorprone.bugpatterns.PreconditionsCheckNotNullPrimitive;
-import com.google.errorprone.bugpatterns.PreconditionsExpensiveString;
-import com.google.errorprone.bugpatterns.PreconditionsInvalidPlaceholder;
-import com.google.errorprone.bugpatterns.PrimitiveArrayPassedToVarargsMethod;
-import com.google.errorprone.bugpatterns.ProtoFieldNullComparison;
-import com.google.errorprone.bugpatterns.ProtoFieldPreconditionsCheckNotNull;
-import com.google.errorprone.bugpatterns.ProtoStringFieldReferenceEquality;
-import com.google.errorprone.bugpatterns.RequiredModifiersChecker;
-import com.google.errorprone.bugpatterns.ReturnValueIgnored;
-import com.google.errorprone.bugpatterns.SelfAssignment;
-import com.google.errorprone.bugpatterns.SelfEquality;
-import com.google.errorprone.bugpatterns.SelfEquals;
-import com.google.errorprone.bugpatterns.SizeGreaterThanOrEqualsZero;
-import com.google.errorprone.bugpatterns.StaticAccessedFromInstance;
-import com.google.errorprone.bugpatterns.StringBuilderInitWithChar;
-import com.google.errorprone.bugpatterns.StringEquality;
-import com.google.errorprone.bugpatterns.SuppressWarningsDeprecated;
-import com.google.errorprone.bugpatterns.TryFailThrowable;
-import com.google.errorprone.bugpatterns.TypeParameterUnusedInFormals;
-import com.google.errorprone.bugpatterns.UnnecessaryStaticImport;
-import com.google.errorprone.bugpatterns.UnnecessaryTypeArgument;
-import com.google.errorprone.bugpatterns.WaitNotInLoop;
-import com.google.errorprone.bugpatterns.WildcardImport;
-import com.google.errorprone.bugpatterns.WrongParameterPackage;
-import com.google.errorprone.bugpatterns.threadsafety.GuardedByChecker;
-import com.google.errorprone.bugpatterns.threadsafety.GuardedByValidator;
-import com.google.errorprone.bugpatterns.threadsafety.LockMethodChecker;
-import com.google.errorprone.bugpatterns.threadsafety.SynchronizeOnNonFinalField;
-import com.google.errorprone.bugpatterns.threadsafety.UnlockMethodChecker;
+import com.google.errorprone.bugpatterns.*;
```

Comment:
At the moment you have .idea set in .gitignore. Can we change this to be something a bit more? It means the intellij xml files that dictate things like code styling etc could be committed